lms/static/scripts/postmessage_json_rpc/server.js

```diff
@@ -112,6 +120,39 @@ export class Server {
     /** @type {WindowProxy} */ (event.source).postMessage(result, event.origin);
   }
 
+  /**
+   * JSON-RPC 2.0 notification requests are distinguished by the lack of an
+   * `id` property.
+   *
+   * @param {MessageEvent} event
+   */
+  _isJSONRPCNotificationRequest(event) {
+    return typeof event.data.id === 'undefined';
```

Comment:
This logic is so simple it might be worth inlining above with a comment explaining the check.